docs/quickstart.html

```diff
@@ -258,14 +258,14 @@
 of the <code>WordCountDemo</code> example code (converted to use Java 8 lambda expressions for easy reading).
 </p>
 <pre>
-KStream<String, Long> wordCounts = textLines
+KTable<String, Long> wordCounts = textLines
     // Split each text line, by whitespace, into words.
     .flatMapValues(value -> Arrays.asList(value.toLowerCase().split("\\W+")))
 
     // Ensure the words are available as message keys for the next aggregate operation.
     .map((key, value) -> new KeyValue<>(value, value))
 
-    // Count the occurrences of each word (message key).
+    // Count the occurrences of each word (message key) and store the results into a table named "Counts".
```

Comment:
Consistent terminology: In Streams we always use the term "record" instead of "message" to distinguish the Streams "layer" from the Kafka "layer".  For example, in Streams we say "record key" and "record value" rather than "message key" and "message value", respectively.